exchange/exchangetest/gdpr-geo-eu.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
Consider renaming to `gdpr-geo-eu-off` to mirror `gdpr-geo-eu-on`.

Reply:
Done